src/pages/instances/InstanceOverviewDeviceList.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdDeviceValue, LxdGPUDevice } from "types/device";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  device: LxdDeviceValue;
+  instance: LxdInstance;
+}
+
+const InstanceOverviewDeviceList: FC<Props> = ({ device, instance }) => {
+  return (
+    <div className="u-truncate">
+      {device.type == "disk" ? (
+        isRootDisk(device as FormDevice) ? (
+          <div title={`Pool: ${device.pool}`}>{`pool: ${device.pool}`}</div>
+        ) : (
+          <>
+            <div title={`Storage Volume: ${device.source}`}>
+              storage volume:
+              <ResourceLink
+                type={"volume"}
+                value={device.source as string}
+                to={`/ui/project/${instance.project}/storage/pool/${device.pool}/volumes/custom/${device.source}`}
+              />
+            </div>
+            <div title={`Pool: ${device.pool}`}>
+              pool:
+              <ResourceLink
+                type={"pool"}
+                value={device.pool}
+                to={`/ui/project/${instance.project}/storage/pool/${device.pool}`}
+              />
+            </div>
+          </>
+        )
+      ) : (
+        ""
+      )}
+      <div title={`UID: ${(device as LxdGPUDevice).pci}`}>
+        {device.type === "gpu" ? `uid: ${device.pci}` : ""}
```

Comment:
why are we using `device.pci` for `uid`? Also we should display "-" instead of ""

Reply:
We shoul display the uid if set, otherwise the id if it is set. We should never have an empty case here.

Reply:
Just checked the jira issue, for `gpu` device, we agreed to display pci address or id. Then the issue here is the `uid:` prefix for the column value. It should display `pci: ` or `id: ` depending on the value that's set on the gpu device